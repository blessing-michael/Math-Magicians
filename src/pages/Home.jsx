import React from 'react';
import '../styles/App.css';

const Home = () => (
  <div className="container">
    <h2 className="welcome">Welcome to Our page!</h2>
    <p className="home-p">
      Are you a Math Magician? Practice a variety of math skills: addition, subtraction,
      multiplication,
      and division using our free webpage. Math magicians allows students to get instant
      solutions to all
      kinds of math problems.
    </p>
    <p className="home-p">
      If you would like to create your own math expressions, here are some symbols that the
      calculator understands:
      <br />

      + (Addition)
      <br />
      - (Subtraction)
      <br />
      * (Multiplication)
      <br />
      / (Division)
    </p>
  </div>
);

export default Home;
