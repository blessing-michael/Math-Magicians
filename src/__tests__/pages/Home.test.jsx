import React from 'react';
import { render } from '@testing-library/react';
import Home from '../../pages/Home';

describe('Home', () => {
  test('Home Page is rendered and matches the snapshot', () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
