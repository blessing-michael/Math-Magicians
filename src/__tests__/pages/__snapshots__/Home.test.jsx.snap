// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Home Page is rendered and matches the snapshot 1`] = `
<div>
  <div
    class="container"
  >
    <h2
      class="welcome"
    >
      Welcome to Our page!
    </h2>
    <p
      class="home-p"
    >
      Are you a Math Magician? Practice a variety of math skills: addition, subtraction, multiplication, and division using our free webpage. Math magicians allows students to get instant solutions to all kinds of math problems.
    </p>
    <p
      class="home-p"
    >
      If you would like to create your own math expressions, here are some symbols that the calculator understands:
      <br />
      + (Addition)
      <br />
      - (Subtraction)
      <br />
      * (Multiplication)
      <br />
      / (Division)
    </p>
  </div>
</div>
`;
